app/tasks/install.php

```diff
@@ -167,6 +167,16 @@
             if(empty($input[$var['name']])) {
                 $input[$var['name']] = $var['default'];
             }
+
+            if ($var['filter'] === 'domainTarget') {
+                if ($input[$var['name']] !== 'localhost') {
+                    Console::warning("\nIf you haven't already done so, set the following record for {$input[$var['name']]} on your DNS provider:\n");
+                    $mask = "%-15.15s %-10.10s %-30.30s\n";
+                    printf($mask, "Type", "Name", "Value");
+                    printf($mask, "A or AAAA", "@", "<YOUR EXTERNAL IP>");
```

Comment:
```suggestion
                    printf($mask, "A or AAAA", "@", "<YOUR PUBLIC IP>");
```

Not sure if 'external' is the correct word here.

Reply:
Yeap Public is better 👍